examples/tcp.rs

```diff
@@ -50,6 +50,57 @@ async fn main() {
         Err(status) => println!("got status during 'add_key' deleted_no_reply: {:?}", status),
     }
 
+    let amount = 1;
+    let increment_key = "increment_key";
```

Comment:
This variable is defined but only used once. You should instead replace every other instance of "increment_key" with the variable instead.